integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/CloudbreakClusterTestConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package com.sequenceiq.it.cloudbreak.newway;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.BeforeSuite;
+
+public class CloudbreakClusterTestConfiguration extends CloudbreakTest {
+    private static final Logger LOGGER = LoggerFactory.getLogger(CloudbreakClusterTestConfiguration.class);
+
+    private static final int MAX_RETRY = 360;
+
+    private static final int SLEEP_TIME = 10000;
+
+    @BeforeSuite
+    public void cleanUpClustersBeforeTest() throws Exception {
+        given(CloudbreakClient.isCreated());
+        when(Stack.getAll());
+        then(Stack.assertThis((stacks, context) -> {
+            stacks.getResponses().stream().forEach(stack -> {
+                try {
+                    given(Stack.request().withName(stack.getName()));
+                    when(Stack.delete(StackAction::deleteWithForce));
+                } catch (Exception e) {
+                    LOGGER.info("Could not delete stack", e);
+                }
+
+            });
+        }));
+        int retryCount = 0;
+        while (!CloudbreakClient.getTestContextCloudbreakClient().apply(getItContext()).getCloudbreakClient().stackV2Endpoint().getPublics().isEmpty()
+                && retryCount++ < MAX_RETRY) {
+            Thread.sleep(SLEEP_TIME);
```

Comment:
Can we use TimeUnit.SECONDS.sleep instead of Thread sleep?

Reply:
yepp